client/components/auth-form.js

```diff
@@ -60,6 +65,50 @@ class AuthForm extends Component {
     loginFunc(email, password).catch(err => console.error(err));
   }
 
+  signInWithGoogle = () => {
+    const googleProvider = new firebase.auth.GoogleAuthProvider();
+    firebase.auth().signInWithPopup(googleProvider).then(result => {
+      // This gives you a Google Access Token. You can use it to access the Google API.
+      var token = result.credential.accessToken;
+      // The signed-in user info.
+      var user = result.user;
+      // ...
+    })
+    // .then(() => {
+    //   history.push('/singleUser')
+    // })
+    .catch(function(error) {
+      // Handle Errors here.
+      var errorCode = error.code;
+      var errorMessage = error.message;
+      // The email of the user's account used.
+      var email = error.email;
+      // The firebase.auth.AuthCredential type that was used.
+      var credential = error.credential;
```

Comment:
I know you copied this stuff from the documentation, but convert `var` to either `let` or `const`. You can also use object destructuring here. `const { errorCode, errorMessage, email, credential } = error;`